resources/build/test/build-bot/trigger-build-bot.test.sh

```diff
@@ -14,7 +14,7 @@ THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
 # Initialize builder, no input parameters needed for these tests
 builder_parse ""
 
-readonly ALL_BUILD_PLATFORMS_SKIP_EXPECTED='[ios]="skip" [web]="skip" [linux]="skip" [common_mac]="skip" [common_windows]="skip" [mac]="skip" [windows]="skip" [common_web]="skip" [android]="skip" [developer]="skip" [common_linux]="skip"'
+readonly ALL_BUILD_PLATFORMS_SKIP_EXPECTED='[ios]="skip" [web]="skip" [linux]="skip" [common_mac]="skip" [common_windows]="skip" [mac]="skip" [windows]="skip" [common_web]="skip" [android]="release" [developer]="skip" [common_linux]="skip"'
```

Comment:
If this is testing all platforms skip, shouldn't android stay "skip"?

Reply:
No, but we should rename the variable. Done.